src/action-constants.js

```diff
@@ -14,6 +14,20 @@
  * limitations under the License.
  */
 
+/**
+ * Key string in an action arguments map for an unparsed object literal string.
+ *
+ * E.g. for the action in <p on="tap:AMP.setState({foo: 'bar'})",
+ * then `args[OBJECT_STRING_ARGS_KEY]` is the string "{foo: 'bar'}".
+ *
+ * The action service delegates parsing of object literals to the corresponding
+ * extension (in the example above, amp-bind).
+ *
+ * @see ./service/action-impl.ActionInfoDef
+ * @const {string}
+ */
+export const OBJECT_STRING_ARGS_KEY = '__AMP_OBJECT_STRING__';
```

Comment:
`RAW_OBJECT_STRING_KEY`?

Reply:
Renamed to `RAW_OBJECT_ARGS_KEY`. Want to keep the fact that this is a key in the "args" dictionary.